yuyo/modals.py

```diff
@@ -907,8 +908,6 @@ async def custom_modal(
 
     __slots__ = ("_ephemeral_default", "_rows", "_tracked_fields")
 
-    _all_static_fields: typing.ClassVar[list[_TrackedField]] = []
-    _all_static_rows: typing.ClassVar[list[hikari.impl.ModalActionRowBuilder]] = []
     _static_fields: typing.ClassVar[list[_TrackedField]] = []
     _static_rows: typing.ClassVar[list[hikari.impl.ModalActionRowBuilder]] = []
```

Comment:
In theory these should be immutable sequences to avoid issues around accidental mutations